### PR TITLE
Improve release script

### PR DESCRIPTION
**Pull request by timowest**

### CODE REVIEW

File: release.sh [hunk 1]
```diff
@@ -18,18 +19,18 @@ doit() {
   git pull
   mvn clean deploy -DskipTests -Dgpg.skip=false
   ./dist.sh
-  # TODO upload
+  ssh root@querydsl.com "mkdir /var/www/html/static/querydsl/$VERSION"
+  scp -r target/dist/* root@querydsl.com:/var/www/html/static/querydsl/$VERSION/
```
Comment (Shredder121) [inline release.sh]: Could you add this command as well?
`cd /var/www/html/static/querydsl && unlink latest && ln -sT $VERSION latest`.

Comment (timowest) [inline release.sh]: done

File: release.sh [hunk 2]
```diff
@@ -18,18 +19,18 @@ doit() {
   git pull
```
Comment (Shredder121) [inline release.sh]: You can also make this `git pull --ff-only` just in case, but not really necessary.

**Comment by Shredder121** [comment]
Creating a pull request is not quite hard, we can do that as well.
https://developer.github.com/v3/pulls/#create-a-pull-request
The only thing that differs is the target (if the version is 4, master, else QUERYDSL_3).

**Comment by timowest** [comment]
Let's do the other github integrations in another PR.

**Pull request merged**